src/Wallabag/CoreBundle/Command/CleanDuplicatesCommand.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace Wallabag\CoreBundle\Command;
+
+use Doctrine\ORM\NoResultException;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Wallabag\CoreBundle\Entity\Entry;
+use Wallabag\UserBundle\Entity\User;
+
+class ExportCommand extends ContainerAwareCommand
+{
+	/** @var OutputInterface */
+	protected $output;
+
+	protected function configure()
+	{
+		$this
+			->setName('wallabag:cleanduplicates')
+			->setDescription('Cleans the database for duplicates')
+			->setHelp('This command helps you to clean your articles list in case of duplicates')
+			->addArgument(
+				'username',
+				InputArgument::OPTIONAL,
+				'User to clean'
+			);
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output)
+	{
+		$this->output = $output;
+
+		$username = $input->getArgument('username');
+
+		if ($username) {
+			try {
+				$user = $this->getUser($username);
+				$this->cleanDuplicates($user);
+			} catch (NoResultException $e) {
+				$output->writeln(sprintf('<error>User "%s" not found.</error>', $input->getArgument('username')));
+
+				return 1;
+			}
+		} else {
+			$users = $this->getDoctrine()->getRepository('WallabagUserBundle:User')->findAll();
+
+			$output->write(sprintf('Cleaning through %d user accounts', count($users)));
+
+			foreach ($users as $user) {
+				$this->cleanDuplicates($user);
+			}
+		}
+		return 0;
+	}
+
+	/**
+	 * @param User $user
+	 */
+	private function cleanDuplicates(User $user)
+	{
+
+		$em = $this->getContainer()->get('doctrine.orm.entity_manager');
+
+		$duplicatesCount = 0;
+		$entries = $user->getEntries();
```

Comment:
Don't you have performance issue with that? Because I guess it'll retrieve the whole entry and associated relation.
Maybe you should better use a custom repository method to retrieve only the entry id and url.
Then, when you need to remove and entry, you can do it using only its id.